Create ui-c8y-1020-24-2-totp-challenge-view-6870-graft-release-cd-7013.md (#2418)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1020-24-2-totp-challenge-view-6870-graft-release-cd-7013.md

---
date: ""

> title: The appearance for the TFA settings has been improved.

Reviewer: ```suggestion
title: Adjusted buttons in the Login screen and TFA settings screen
```

> title: totp challenge view (#6870) [GRAFT][release/cd] (#7013)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Add TOTP challenge view
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57801
version: 1020.24.2
---

> totp challenge view (#6870) [GRAFT][release/cd] (#7013)

Author: The Claude AI suggested to use the following body:
```suggestion
To enhance the security of user accounts, a new TOTP (Time-based One-Time Password) challenge view has been added. When enabled, users will be prompted to enter a TOTP code in addition to their password during the login process. This change introduces an extra layer of authentication, helping to protect user accounts from unauthorized access. Existing users will need to set up TOTP for their accounts to take advantage of this enhanced security feature.
```

> Buttons and control sizes have been adjusted to fit the login screen and the TFA settings screen available in the user settings.

Reviewer: ```suggestion
Buttons and control sizes have been adjusted to fit the Login screen and the TFA settings screen available in the user settings.
```
